it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -53,7 +53,7 @@ class TraderServicesFrontendISpec
     }
```

Comment:
Failing test on this page: <img width="710" alt="image" src="https://user-images.githubusercontent.com/55190161/96109857-09cb2d80-0ed7-11eb-9898-a36621baf340.png">